src/main/webapp/resources/js/pages/cart/index.js

```diff
@@ -20,21 +21,19 @@ See: https://webpack.js.org/guides/public-path/#on-the-fly
  */
 __webpack_public_path__ = setBaseUrl(`dist/`);
 
-const store = getStore(
-  {},
-  {
-    empty,
-    removeSampleFromCart,
-    removeProjectFromCart,
-    loadFullCart,
-  }
-);
+const store = configureStore({
+  reducer: {
+    [cartApi.reducerPath]: cartApi.reducer,
+    [projectsApi.reducerPath]: projectsApi.reducer,
+    share: shareReducer,
```

Comment:
This can be removed as its no longer used

Reply:
Removed